action.yml

```diff
@@ -67,7 +67,8 @@ runs:
       working-directory: .js-client/artifact
       shell: sh
 
-    - run: yarn publish --patch --access restricted
+    # Fetch latest remote version && publish new version
+    - run: yarn version --new-version "`npm show @coinpanel/user-secrets-api version`" && yarn publish --patch --access restricted
```

Comment:
A hard-coded value `@coinpanel/user-secrets-api` can be replaced with `${{ inputs.npm-package-name }}`.